.github/workflows/build_and_test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Upload Windows Client
         uses: actions/upload-artifact@v2
         with:
-          name: unsigned-windows-client
+          name: debug-windows-client
```

Comment:
I wonder if we should call this installer-windows-debug, since this is actually the installer of the client.

Reply:
client-installer? this is just the case for windows, right